README.md

```diff
@@ -2,33 +2,38 @@
 ![Mixer](doc/logo.png)
 
 [![GoDoc](https://godoc.org/github.com/istio/mixer?status.svg)](https://godoc.org/github.com/istio/mixer)
-[![Build Status](https://testing.istio.io/buildStatus/icon?job=mixer/postsubmit)](https://testing.istio.io/job/mixer/)
-[![Go Report Card](https://goreportcard.com/badge/github.com/istio/mixer)](https://goreportcard.com/report/github.com/istio/mixer)
+[![Go Report
+Card](https://goreportcard.com/badge/github.com/istio/mixer)](https://goreportcard.com/report/github.com/istio/mixer)
 [![codecov.io](https://codecov.io/github/istio/mixer/coverage.svg?branch=master)](https://codecov.io/github/istio/mixer?branch=master)
 
-Mixer provides the foundation of the Istio service mesh
-design. It is responsible for insulating Envoy and
-Istio-based services from details of the current execution
-environment, as well as to implement the control policies
-that Istio supports.
+Mixer enables extensible policy enforcement and control within the Istio service
+mesh. It is responsible for insulating the sidecar proxy (Envoy) from details of
```

Comment:
More than just the sidecar, right? It's also for ingress.

Reply:
removed "sidecar".